src/Microsoft.Azure.SignalR.Management/HubContext/GroupManagerAdapter.cs

```diff
@@ -10,14 +10,18 @@ namespace Microsoft.Azure.SignalR.Management
     internal class GroupManagerAdapter : GroupManager
     {
         private readonly IGroupManager _groupManager;
+        private readonly IHubLifetimeManager _lifetimeManager;
 
-        public GroupManagerAdapter(IGroupManager groupManager)
+        public GroupManagerAdapter(IGroupManager groupManager, IHubLifetimeManager lifetimeManager)
```

Comment:
Remove `IGroupManager` and implements all the methods with `IHubLifetimeManager`?

Reply:
updated